app/main.py

```diff
@@ -1 +1,24 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    sepp_command = command.split()
+
+    source = sepp_command[1]
+    destination = sepp_command[2]
+
+    path_to_move = destination.split("/")
+
+    new_dir = ""
+
+    if path_to_move:
+        for path in path_to_move[:-1]:
+            new_dir += path + "/"
+            if not os.path.exists(new_dir):
```

Comment:
The current implementation does not handle the case where a directory already exists. Attempting to create an existing directory will lead to an error.